index.js

```diff
@@ -20,23 +20,25 @@ const core = require('@actions/core');
 const execSync = require('child_process').execSync;
 
 try {
-  // check it CycloneDX is installed
+  // check if CycloneDX is installed
   try {
     execSync('cyclonedx-py -h');
   } catch (error) {
     console.log('Installing CycloneDX...');
-    let consoleOutput = execSync('pip3 install cyclonedx-bom', { encoding: 'utf-8' });
+    let consoleOutput = execSync('pip3 install cyclonedx-bom>=1.4.0,<=3.11.2', { encoding: 'utf-8' });
```

Comment:
@jkowalleck should I make this <=4 to cover the upcoming v4.

Reply:
yes, please! :+1: 

`cyclonedx-bom` is semantic-versioning aware.
there will be no breaking changes in `3.x`, so `<4` is appropriate.

Reply:
fixed that